nixos/tests/ntfy-sh.nix

```diff
@@ -0,0 +1,20 @@
+import ./make-test-python.nix {
+
+  nodes.machine = { pkgs, ... }: {
+    services.ntfy-sh.enable = true;
+
+    environment.systemPackages = [ pkgs.jq ];
+  };
+
+  testScript = ''
+    msg = "Test notification"
+
+    machine.wait_for_unit("multi-user.target")
+
+    machine.succeed(f"curl -d '{msg}' localhost:80/test")
+
+    notif = machine.succeed("curl -s localhost:80/test/json?poll=1 | jq '.message'")
```

Comment:
Maybe this should be changed to actually parsing the output with Python's JSON module? Maybe not, idk, just a thought.

Reply:
@Tom-Hubrecht @arjan-s Any thoughts on this? Last thing to discuss before it's good to go :)

(To be clear, I'm fine with either really, but would just appreciate feedback. Thanks!)

Reply:
Ahh, I missed the fact that `testScript` is actual Python code. In that case, yes, I would parse it using the built-in JSON module.

Reply:
Done, it is better to parse the json in python indeed